trifecta.cabal

```diff
@@ -112,18 +106,6 @@ library
       fail                      == 4.9.*,
       semigroups                >= 0.8.3.1 && < 1
 
-test-suite doctests
-  type:              exitcode-stdio-1.0
-  main-is:           doctests.hs
-  ghc-options:       -Wall -threaded
-  x-doctest-options: -fobject-code
```

Comment:
I recall that with `doctest`, we had to enable `-fobject-code` directly to support the use of unboxed tuples in `trifecta`. I know that `cabal-docspec` behaves similarly to as if you had `-fobject-code` enabled, but does it actually need to enable the flag here in order to function properly?

Reply:
Recall, `cabal-docspec` loads already compiled library, it doesn't interpret the library source. Thus it's always, kind of `-fobject-code`.

Reply:
TL;DR with `cabal-docspec` you don't need `--fast` nor `-fobject-code`.